go/cmd/mysqlctld/cli/mysqlctld.go

```diff
@@ -84,8 +85,9 @@ func init() {
 	Main.Flags().IntVar(&mysqlPort, "mysql_port", mysqlPort, "MySQL port")
 	Main.Flags().Uint32Var(&tabletUID, "tablet_uid", tabletUID, "Tablet UID")
 	Main.Flags().StringVar(&mysqlSocket, "mysql_socket", mysqlSocket, "Path to the mysqld socket file")
-	Main.Flags().DurationVar(&waitTime, "wait_time", waitTime, "How long to wait for mysqld startup or shutdown")
+	Main.Flags().DurationVar(&waitTime, "wait_time", waitTime, "How long to wait for mysqld startup")
```

Comment:
Fun fact, this config value was not used at all for shutdown, only for startup so call it out as such. Also we want these to really be separate.

Reply:
What do you think of fixing this to actually be used during both startup and shutdown? Then you could just pass the desired higher value. Are there downsides to increasing the startup wait time?

Reply:
@deepthi I think they are very different concepts? How long you want to wait for startup can be very different I think than for shutdown.

Especially in the case that is relevant here for disabling fast shutdown on upgrades, you want to wait a lot longer (say 30 minutes) for shutdown, but waiting that long for startup is entirely unneeded and likely not desired since you want to know earlier if you have a problem starting up MySQL. 

That's why I think they must be separate values really and assuming the same value for both doesn't make sense.

Reply:
But it's effectively a wait time for the mysqlctl client command / mysqlctld RPC, no? In that case I also think we should just have a single wait timeout flag. I'm OK with adding another flag, if it's truly necessary. Looks like this is also relevant for other programs though such as vttablet, so having an additional flag there is required and there's IMO less reason to avoid that for mysqlctl[d].

Reply:
> But it's effectively a wait time for the mysqlctl client command / mysqlctld RPC, no?

Not really I think. The only operation that uses it is the start command. Also I would not classify it as an RPC, because this is for the internal "starting MySQL" phase and it's not used for RPC to `mysqlctdl`. 

> I'm OK with adding another flag, if it's truly necessary.

I don't see a way without it. Startup is a fundamentally different thing from shutdown and therefore I think they need separate controls.